internal/controller/ansibleRun/ansibleRun.go

```diff
@@ -448,6 +448,10 @@ func (c *external) handleLastApplied(ctx context.Context, lastParameters *v1alph
 	isLastSyncOK := (desired.GetCondition(xpv1.TypeSynced).Status == v1.ConditionTrue)
 
 	if isUpToDate && isLastSyncOK {
+		desired.SetConditions(xpv1.Available())
```

Comment:
When setting `Ready` condition right after a successful run, various transient errors can happen preventing this status update, and, potentially, leaving the condition `False` forever. This ensures `Ready` is set even if such errors occur